tests/lib.rs

```diff
@@ -876,3 +876,326 @@ Error codes:
         );
     }
 }
+
+#[test]
+fn redact_arg_values_no_args() {
+    #[derive(FromArgs, Debug)]
+    /// Short description
+    struct Cmd {
+        #[argh(option)]
+        /// a msg param
+        msg: Option<String>,
+    }
+
+    let actual = Cmd::redact_arg_values(&["program-name"], &[]).unwrap();
```

Comment:
nit: I really appreciate the change to using `program-name` here. Would you be willing to replace the other `<<<arg0>>>` literals in this file with `program-name`?

Reply:
done!